client/sites/components/sites-dataviews/actions.tsx

```diff
@@ -267,13 +267,10 @@ export function useActions( {
 	viewType: 'list' | 'table' | 'grid';
 } ): Action< SiteExcerptData >[] {
 	const { __ } = useI18n();
-
 	const localizeUrl = useLocalizeUrl();
-
-	const dispatch = useReduxDispatch();
-
+	const dispatch = useDispatch();
 	const queryClient = useQueryClient();
-	const reduxDispatch = useReduxDispatch();
```

Comment:
This is a drive-by change that I noticed when modifying the component that calls `useActions`. We don't need to call `useDispatch` twice! Probably some copy&paste or AI generation caused this.